### DISCUSSION TITLE
Use SD v1.x and 2.x at the same time...

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#1587] [Ideas]
Français 

J'ai une suggestion, je suis sur la version 1.5 (sur google colab) et pas très fan de la 2.x d'autant plus que pour la tester il faut actuellement supprimer le dossier SD et tout reconfigurer quand on passe d'une version 1x à une 2x... a chaque fois pas pratique...
mon idée est simple... pourquoi ne pas tout simplement faire 2 dossier SD séparés, par exemple un SD1 avec les modèles compatibles 1.x et un dossier SD2 avec la même chose mais pour les modèles 2.x et on passerait de l'un a l'autre en sélectionnant la version qu'on veut utiliser dans automatic 1111 sans rien supprimer,  réinstaller etc etc...  

(note : soit on passe de l'un a l'autre avec deux dossier SD1 et SD2 , soit on pourrait avoir un dossier SD1 et SD (configuration avec la version 2.x activée et si on sélectionne 1.5, A1111 renomme  SD en SD2 et SD1 en SD ... mais 2 sans renommage c'est plus simple...) 

bonne idée ou pas ? 

Anglais 

I have a suggestion, I'm on version 1.5 (on google colab) and not a big fan of the 2.x especially since to test it you have to delete the SD folder and reconfigure everything when you go from a 1x to a 2x version... each time not practical... my idea is simple... why not simply make 2 separate SD folders, for example one SD1 with 1.x compatible models and one SD2 folder with the same thing but for 2.x models and we would switch from one to the other by selecting the version we want to use in automatic 1111 without deleting anything, reinstalling etc etc...  

(note: either we go from one to another with two folders SD1 and SD2, or we could have a folder SD1 and SD (configuration with version 2.x activated and if we select 1.5, A1111 renames SD to SD2 and SD1 to SD ... but 2 without renaming it's simpler ...)

good idea or not ?

**Comment by TheLastBen** [comment]
Mais il est tout à fait possible d'utiliser les deux versions au même temps, maintenant A1111 détecte automatiquement la version.

**Comment by darth2602** [comment]
quand la version 2.0 est arrivée ça ne fonctionnait pas, il fallait supprimer le dossier SD choisir 2.0 a la place de 1.5 et relancer .... donc maintenant on passe de l'une a l'autre sans rien supprimer ?  il suffit de retélécharger le modèle  ?  par contre il n'y a plus l'option 
pour télécharger le modèle inpaint quelle que soit la version (et j'ai du le faire a la main car même en demandant de retélécharger, sans changer de version, ça ne le faisait pas....   c'est peu être normal, je ne sais pas, peut être que le modèle inpaint ets téléchargé seulelment quand on l'utilise (la première fois) ?

**Reply by TheLastBen** [reply]
Tu peux change la version du modèle sans rien supprimer, et le inpanting modèle est dans la liste avec le v1.5 et v2, vérifie que tu as la dernière version du colab

**Comment by darth2602** [comment]
oui, je retélécharge ma version  tout les jours , mais comme quand la version 2 est sortie ça ne marchait pas comme ça , j'a essayé une ou deux fois et je suis resté a la 1.5.... quand j'ai eu le problème tu m'a dit de supprimer le dossier SD a chaque fois pour passer d'une version a l'autre, mais si maintenant tout est ok, c'est cool.... ;-)
